opentelekomcloud/resource_opentelekomcloud_rds_instance_v1_test.go

```diff
@@ -0,0 +1,130 @@
+package opentelekomcloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/huaweicloud/golangsdk/openstack/rds/v1/instances"
+)
+
+// PASS
+func TestAccRDSV1Instance_basic(t *testing.T) {
+	var instance instances.Instance
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRDSV1InstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: TestAccSInstanceV1Config_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRDSV1InstanceExists("opentelekomcloud_rds_instance_v1.instance", &instance),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_rds_instance_v1.instance", "status", "ACTIVE"),
```

Comment:
You probably should read more of the output attributes here if possible to confirm they are working as expected.